### ISSUE TITLE
Shadows with opacity are not applied

### ISSUE THREAD

**Issue by aydinfatih** [johynpapin/flutter_inset_shadow#1]
I add opacity to the shadow color but it tints without opacity. Can you help with this issue?

```

Container(
      width: width,
      height: height,
      padding: padding,
      decoration: BoxDecoration(
        boxShadow: [
          BoxShadow(
            offset: Offset.fromDirection(4, 4) ,
            color: Colors.black.withOpacity(.05),
            blurRadius: 0,
            spreadRadius: 10,
            inset: true,
          ),
        ],
      ),
      child: child,
    )
```
![0c2f6cd9-5c50-4fae-9253-2949a00db6e7](https://user-images.githubusercontent.com/14280894/162429852-e6b4595c-0b0e-4684-bf30-482dcf1e2a06.jpeg)

**Comment by aydinfatih** [comment]
Fixed
#2

**Comment by ghost** [comment]
Hello ! Thank you very much for this PR, I am merging it immediately.

**Comment by ghost** [comment]
I just published version 1.0.8 on pub.dev, feel free to upgrade.
